src/Appwrite/Extend/Exception.php

```diff
@@ -164,6 +164,8 @@ class Exception extends \Exception
     public const PROJECT_RESERVED_PROJECT          = 'project_reserved_project';
     public const PROJECT_KEY_EXPIRED               = 'project_key_expired';
 
+    public const PROJECT_DEFAULT_TEMPLATE_DELETION = 'project_default_template_deletion';
```

Comment:
PROJECT_TEMPLATE should be the namespace.

Reply:
Updated